clang/test/Misc/pragma-attribute-supported-attributes-list.test

```diff
@@ -136,14 +135,10 @@
 // CHECK-NEXT: Pointer (SubjectMatchRule_record_not_is_union)
 // CHECK-NEXT: ReleaseHandle (SubjectMatchRule_variable_is_parameter)
 // CHECK-NEXT: RenderScriptKernel (SubjectMatchRule_function)
-// CHECK-NEXT: ReqdWorkGroupSize (SubjectMatchRule_function)
```

Comment:
I don't have OpenCL background, so I cannot say is correct change or not, but this particular attribute `ReqdWorkGroupSize` is OpenCL specific and it is added not by us.
Suggest you go through community with change for this attribute first.
I think you will get answers on your questions regarding this attribute.

Reply:
Ok, lets stick with leaving reqd_work_group_size attribute alone.